src/mesh/RTT_Draco_Mesh_Reader.hh

```diff
@@ -34,14 +34,15 @@ private:
 
 public:
   //! Constructor
-  DLL_PUBLIC_mesh explicit RTT_Draco_Mesh_Reader(const std::string filename_);
```

Comment:
👍 Nice to see these disappearing slowly.  Alex and have a plan to remove all the remaining entries after the release. (except for the handful that are still needed.)

Reply:
No problem - good to know!